### PR TITLE
Fix export: preprocess_fn returns 6 variables instead of 5

### PR DESCRIPTION
**Pull request by davidblom603**
Refer to https://github.com/google-research/deeplab2/blob/main/data/preprocessing/input_preprocessing.py#L350
That method returns 6 variables.

### CODE REVIEW

**Comment by davidblom603** [comment]
@googlebot I signed it!

File: export_model.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def __call__(self, input_tensor: tf.Tensor) -> MutableMapping[Text, Any]:
       processed_image = tf.concat(
           [processed_image, processed_prev_image], axis=2)
     else:
-      (resized_image, processed_image, _, _, _) = self._preprocess_fn(
+      (resized_image, processed_image, _, _, _, _) = self._preprocess_fn(
```
Comment (aquariusjay) [inline export_model.py:129]: Thanks for the fix, davidblom603!
Could you please also help update Line125?
Thanks,

Comment (davidblom603) [reply export_model.py:129]: Done

**Review by aquariusjay — COMMENTED** [review]
Looks good on my side.
Thanks, davidblom603!

**Review by ClaireXie — APPROVED** [review]
Thanks David!

**Pull request merged**